packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -48,17 +47,7 @@ const MessageContentAsText = (message: ChatMessage, liveAuthorIntro: string): JS
   return (
     <div data-ui-status={message.status}>
       <LiveMessage message={`${message.mine ? '' : liveAuthor} ${message.content}`} aria-live="polite" />
-      <Linkify
-        componentDecorator={(decoratedHref: string, decoratedText: string, key: number) => {
-          return (
-            <Link target="_blank" href={decoratedHref} key={key}>
-              {decoratedText}
-            </Link>
-          );
-        }}
-      >
-        {message.content}
-      </Linkify>
+      <Linkify>{message.content}</Linkify>
```

Comment:
I think you need to specify target still: https://linkify.js.org/docs/options.html#target

Reply:
Great catch. I reverted back the old code here so that a new tab is opened when links are clicked. Thanks.
https://user-images.githubusercontent.com/79475487/165403162-f249666e-9683-4100-acbd-7d8fa1a9b128.mp4